cmd/esbuild/main.go

```diff
@@ -46,7 +46,7 @@ Options:
   --jsx-factory=...     What to use instead of React.createElement
   --jsx-fragment=...    What to use instead of React.Fragment
   --loader:X=L          Use loader L to load file extension X, where L is
-                        one of: js, jsx, ts, tsx, json, text, base64, dataurl
+                        one of: js, jsx, ts, tsx, json, text, base64, url, dataurl
```

Comment:
This feature seems similar to the Webpack [file-loader](https://webpack.js.org/loaders/file-loader/) plugin, so I think people will understand the name `file` better than `url`. There is also a Rollup [url](https://github.com/rollup/plugins/tree/master/packages/url) plugin which does something different (it does what `dataurl` does in esbuild), so that may also be confusing for people coming from Rollup.

Reply:
that's true! url-loader seems to have more specific feature in webpack world https://webpack.js.org/loaders/url-loader/ we could use `url` to automatically switch between `dataurl` and `file` in the future i guess. i'll rename this to `file`